deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -15,12 +17,14 @@ services:
       retries: 3
 
   alertmanager:
-    image: signoz/alertmanager:0.5.0
+    image: signoz/alertmanager:0.6.0
     volumes:
-      - ./alertmanager.yml:/prometheus/alertmanager.yml
+    # we no longer need the config file as query services delivers 
+    # the required config now 
+    # - ./alertmanager.yml:/prometheus/alertmanager.yml
       - ./data/alertmanager:/data
     command:
-      - '--config.file=/prometheus/alertmanager.yml'
+      - '--queryService.url=http://localhost:8080'
```

Comment:
`query-service` won't be accessible at `localhost`. Rather the url should be `http://query-service:8080` IMO

Reply:
Fixed.